docs/source/documentation/r_scripts.rst

```diff
@@ -0,0 +1,117 @@
+How-to use R scripts with Substra
+=================================
+
+SubstraFL is a library made for working in Python, but Substra is flexible enough to accommodate running tasks in other programming languages.
+This how-to guide explains how to run scripts written in R with Substra.
+This uses the low-level interface of Substra and requires writing more boilerplate code than using the high-level interface of SubstraFL.
+If you are not familiar with the Substra low-level library, you should read the
+:ref:`Substra introductory example <Titanic example>` first.
+
+.. caution:: This guide provides an easy to run some scripts in another language. The scripts are wrapped up in a Python process, so performances might be limited.
```

Comment:
I like this caution box, should be more explicit and mention that it won't be multithreaded?